src/frontend/packages/lib_video/src/components/live/Teacher/Wrapper/Controls/TeacherLiveLifecycleControls/index.tsx

```diff
@@ -68,23 +68,23 @@ export const TeacherLiveLifecycleControls = ({
   let content;
   if (!canStartStreaming) {
     content = (
-      <Paragraph
-        margin={firstItemMargin}
+      <Text
```

Comment:
Type p ?

Reply:
Same, it is an inline text, should be a span.